### ISSUE TITLE
Random test failure chunks_delay_tracker::test::test_timestamps

### ISSUE THREAD

**Issue by pmnoxx** [near/nearcore#5251]
We see those random failures happen a lot.
```

[2021-11-14T19:04:35Z] running 7 tests
--
  | [2021-11-14T19:04:35Z] test sync::test::test_get_locator_heights ... ok
  | [2021-11-14T19:04:35Z] test chunks_delay_tracker::test::test_blocks_ahead_of_head ... ok
  | [2021-11-14T19:04:35Z] test chunks_delay_tracker::test::test_timestamps ... FAILED
  | [2021-11-14T19:04:35Z] test sync::test::test_sync_headers_fork ... ok
  | [2021-11-14T19:04:35Z] test sync::test::test_block_sync ... ok
  | [2021-11-14T19:04:35Z] test sync::test::test_block_sync_archival ... ok
  | [2021-11-14T19:04:47Z] test sync::test::test_slow_header_sync ... ok
  | [2021-11-14T19:04:47Z]


```

https://buildkite.com/nearprotocol/nearcore/builds/6683#36f0677f-f2e8-44e8-8d1d-7650a7a2d568

**Comment by pmnoxx** [comment]
@bowenwang1996 @mm-near @matklad Any ideas?

**Comment by bowenwang1996** [comment]
Not sure. Assigning it to @nikurt since he wrote the test

**Comment by pmnoxx** [comment]
@nikurt All metrics inside `metrics`, for example, `metrics::CHUNKS_RECEIVING_DELAY_US` are static variables.
Those tests are flawed, they will continue failing, because there may be another thread accessing those values during the test.

The best course of action would be to make `metrics` a shared object between multiple threads or maybe we should disable those tests for now.

**Comment by pmnoxx** [comment]
Btw, I'm replacing old style of `metrics` with a new one. See https://github.com/near/nearcore/pull/5265 for details.
It may be a good idea to do something similar.

**Comment by pmnoxx** [comment]
@nikurt Let's ignore those tests for tests for now https://github.com/near/nearcore/pull/5272.
They have blocked many PRs from being merged.

**Comment by pmnoxx** [comment]
Closes by #5272
